beetsplug/bpd/gstplayer.py

```diff
@@ -139,27 +157,36 @@ def run(self):
         Call this function before trying to play any music with
         play_file() or play().
         """
+
         # If we don't use the MainLoop, messages are never sent.
-        gobject.threads_init()
 
         def start():
-            loop = gobject.MainLoop()
+            loop = GObject.MainLoop()
```

Comment:
Modern GObject apparently recommends `GLib.MainLoop()` instead. Also in this PR: https://github.com/beetbox/audioread/pull/30/files